docs/topics/sealed-classes.md

```diff
@@ -2,7 +2,7 @@
 
 _Sealed_ classes and interfaces represent restricted class hierarchies that provide more control over inheritance.
 All direct subclasses of a sealed class are known at compile time. No other subclasses may appear outside
-a module within which the sealed class is defined. For example, third-party clients can't extend your sealed class in their code.
+the module or package within which the sealed class is defined. For example, third-party clients can't extend your sealed class in their code.
```

Comment:
```suggestion
the module and package within which the sealed class is defined. For example, third-party clients can't extend your sealed class in their code.
```
I think we need to use "and" here so that it doesn't seem like it's possible within the same package but using a different module.